oshdb/src/main/java/org/heigit/bigspatialdata/oshdb/util/byteArray/FastByteArrayOutputStream.java

```diff
@@ -0,0 +1,71 @@
+package org.heigit.bigspatialdata.oshdb.util.byteArray;
```

Comment:
was this adapted/copied over from [`it.unimi.dsi.fastutil.io.FastByteArrayOutputStream`](https://github.com/vigna/fastutil/blob/master/src/it/unimi/dsi/fastutil/io/FastByteArrayOutputStream.java)? if yes, we should/must state that somewhere.

Reply:
but don't we have this library as a dependency anyway? what's the use for this class here?